### DISCUSSION TITLE
News Cache [2023-05-22]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#2558] [Announcements]
### 🚢  Ships

- [Improved experience for moving between your teams and projects](https://vercel.com/changelog/improved-experience-for-moving-between-your-teams-and-projects)
- [Visual Editing can now be used with Builder.io](https://vercel.com/changelog/visual-editing-can-now-be-used-with-builder-io)
- [New Next.js Docs Page: Project Structure](https://t.co/LDTJGtklFa)

### 💡 Inspiration

- [Vercel + Sanity: Innovating on a faster, more collaborative Web](https://vercel.com/blog/vercel-sanity-innovating-on-a-faster-collaborative-web)

### 📚 Resources

- [Node.js 14 and 16 are being deprecated in August 2023](https://vercel.com/changelog/node-js-14-and-16-are-being-deprecated)
- [Integrate Remix session storage with your Vercel KV database](https://vercel.com/changelog/integrate-remix-session-storage-with-your-vercel-kv-database)
- [Use Vercel KV with Nuxt](https://twitter.com/steph_dietz_/status/1659258363165790208?s=20)
- [Next.js underscore routing trick](https://twitter.com/lydiahallie/status/1659226747110412289?s=20)
- [New Next.js Prisma Postgres Auth Starter](https://twitter.com/steventey/status/1658904575045410818?s=20)

**Comment by yorunning** [comment]
I think that the rendering documentation of Next.js should include an explanation of whether page rendering is SSR or SSG when using a combination of `static`/`dynamic` routes and `cache: 'force-cache'` / `cache: 'no-no-store'`. This part can be confusing for beginners learning Next.js 13, and they may need to refer back to the /page routes to fully understand it.

**Reply by amyegan** [reply]
Thanks for sharing this feedback. I passed it along to the documentation team 🙇
